docs/guide/config_repository.md

```diff
@@ -89,6 +101,12 @@ ezplatform:
             repository: second_repository
 ```
 
+```
+# .env.local
+
+SECOND_DATABASE_URL=otherdb://otheruser:otherpasswd@otherhost:3306/otherdbname?serverVersion=mariadb-10.3.0
```

Comment:
I would either skip the port or state `:otherdbport`, the same goes for `mariadb-10.3.0`.
But that's just minor nitpick, should be known to someone who even tries have multiple connections to different instances.

Reply:
Okay, changed into `otherdbport` and `otherdbserversion`